assets/js/modules/search-console/components/dashboard/DashboardClicksWidget.js

```diff
@@ -52,10 +52,12 @@ function DashboardClicksWidget() {
 		const isDomainProperty = select( STORE_NAME ).isDomainProperty();
 		const referenceSiteURL = untrailingslashit( select( CORE_SITE ).getReferenceSiteURL() );
 
+		const dateRange = select( CORE_USER ).getDateRangeDates( { compare: true, offsetDays: 1 } );
```

Comment:
Can you add a `DATE_RANGE_OFFSET` constant to `assets/js/modules/search-console/datastore/constants.js` and use that in all these locations? This way we put the offset for this module in one place, similarly how it was done for AdSense and Analytics in #2459.

Reply:
Updated and I've added the value as 2 until #2522 is implemented.